packages/jellyfish-api-core/src/category/oracle.ts

```diff
@@ -106,3 +124,10 @@ export interface UTXO {
   txid: string
   vout: number
 }
+
+export interface OracleTokenPrice {
+  token: string
+  currency: string
+  amount: number
+  timestamp: number
```

Comment:
For your remaining PR do document this in detail, I feel like this can be prone to error in the long run.

e.g.

```ts
  /**
   * @exampe 0.00 
   */
  amount: number
  /**
   * Formatted '' encoded
   * @exampe 133 is ...
   */
  timestamp: number
```